src/_P001_Switch.ino

```diff
@@ -674,7 +674,7 @@ boolean Plugin_001(uint8_t function, struct EventStruct *event, String& string)
       // WARNING: don't read "globalMapPortStatus[key]" here, as it will create a new entry if key does not exist
 
 
-      if (command.equals(F("inputswitchstate"))) {
+      if (equals(command, F("inputswitchstate"))) {
```

Comment:
This command has been deprecated for years already, I think this can be removed, as it doesn't work anymore.